### ISSUE TITLE
Sporadically failing firestore tests

### ISSUE THREAD

**Issue by danielpeintner** [eclipse/thingweb.node-wot#755]
The CI fail now sometimes w.r.t. firestore.
@hidetak may I ask you to take a look. Not sure if it is just a timeout issue because Windows seems to work always and it keeps failing on MacOS or Linux

https://github.com/eclipse/thingweb.node-wot/runs/6418368671?check_suite_focus=true

```
  1 failing

  1) Firestore client basic test implementation
       "before each" hook: before for "[client] action about string":
     Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/thingweb.node-wot/thingweb.node-wot/packages/binding-firestore/test/firestore-client-basic-test.ts)
      at listOnTimeout (node:internal/timers:559:17)
      at processTimers (node:internal/timers:502:7)
```

**Comment by hidetak** [comment]
@danielpeintner 

I have created the following pull request.

https://github.com/eclipse/thingweb.node-wot/pull/757

We apologize for the inconvenience and ask for your confirmation.

**Comment by danielpeintner** [comment]
Thanks, I merged #757. Let's hope it resolves the issue.
